package.json

```diff
@@ -35,7 +35,8 @@
     "mime": "^3.0.0",
     "node-fetch": "^2.6.5",
     "node-webpmux": "^3.1.0",
-    "puppeteer": "^13.0.0"
+    "puppeteer": "^13.0.0",
+    "link-preview-js": "^3.0.4"
```

Comment:
We shouldn't use this.
1 - it is fairly easy to do something basic.
2 - We should use the waweb way of doing
3 - The lib is no longer maintained

Reply:
1. It is the way used by Baileys
2. WhatsApp Way of doing is a React component and some weird Web APIs, and images/info is gotten from the socket using Sync States (you send a sync request for the server to scrape the page for you), trust me I tried for 2 hours
3. That doesn't really matter as the standards that the library uses aren't about to change anytime soon, if so, the API opens access to headers and other configurations